ocaml/idl/datamodel_host.ml

```diff
@@ -2071,6 +2071,9 @@ let t =
         ; field ~qualifier:DynamicRO ~in_product_since:"1.313.0" ~ty:Bool
             "tls_verification_enabled" ~default_value:(Some (VBool false))
             "True if this host has TLS verifcation enabled"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:String
```

Comment:
Now that this is a new field, you can use the `DateTime` type rather than `String`.